src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -96,72 +144,174 @@ export const MultiSelect: FC<MultiSelectProps> = ({
         onSelectionChange(Array.from(keySet));
     };
 
-    const summarizedLabel = [activeItemKeys.length, `option${activeItemKeys.length > 1 ? 's' : ''}`, 'selected'].join(
-        ' ',
-    );
+    const handleSpacebarToggle = (e: KeyboardEvent<HTMLDivElement>) => {
+        if (e.code === 'Space' && (filterInputRef.current !== document.activeElement || !filterable)) {
+            toggleOpen();
+        }
+    };
+
+    const getDecoratorClasses = () => {
+        if (emphasis === MultiSelectEmphasis.Weak) {
+            return hasSelectedItems ? '' : 'tw-text-text-weak';
+        } else {
```

Comment:
you can remove the `else` statement here and just use `return ''` since it will return inside the `if`

Reply:
It is not about saving two lines of code. It is about avoiding future unreachable code if any written after the `else`. And It would be ok if you decided not to change it now. I know there is more than one solution for a problem, and it is great when we use this space for at least pointing them and learning from each other. I'm sorry if it looked I was only trying to save two lines of code.